src/utils/narrow.js

```diff
@@ -537,3 +537,24 @@ export const getNarrowForReply = (message: Message | Outbox, ownUserId: UserId)
     return topicNarrow(streamName, message.subject);
   }
 };
+
+/**
+ * Answers, this question, "What should be the right accessibilityLabel for
+ * send message button?"
+ *
+ * This function only provide labels for stream, topic and pm narrow.
+ * For other narrows, there is no point of label.
+ */
+export function narrowUiDescription(narrow: Narrow, _: GetText): string {
```

Comment:
Ah, looking again at the other things in this file: I think a name like `uiDescriptionOfNarrow` would be better. (See that there are several functions that end in `OfNarrow` and take a `Narrow`.